cmd/kubelet/app/options/options.go

```diff
@@ -265,6 +266,18 @@ func ValidateKubeletFlags(f *KubeletFlags) error {
 		klog.Warningf("unknown 'kubernetes.io' or 'k8s.io' labels specified with --node-labels: %v", unknownLabels.List())
 		klog.Warningf("in 1.15, --node-labels in the 'kubernetes.io' namespace must begin with an allowed prefix (%s) or be in the specifically allowed set (%s)", strings.Join(kubeletapis.KubeletLabelNamespaces(), ", "), strings.Join(kubeletapis.KubeletLabels(), ", "))
 	}
+	if len(f.KubeConfig) > 0 {
+		if _, err := os.Stat(f.KubeConfig); err == nil {
+			// if the destination already exists, do nothing
+			return nil
+		} else if os.IsPermission(err) {
+			// if we can't access the file, skip it
```

Comment:
If I can make a suggestion, please consider logging this perhaps at `klog.Debug` (or its equivalent) since the whole idea is to point out reasons kubelet isn't behaving like one expects

Reply:
OK , i will add a warning log here